pkg/storage/queue.go

```diff
@@ -754,6 +754,35 @@ func (bq *baseQueue) processReplica(queueCtx context.Context, repl *Replica) err
 	return nil
 }
 
+type benigner interface {
```

Comment:
I'd call this interface `benignError` and the struct `benignWrapper`. (I can spell "benign" instead of "begin", but I can't verb it)